### PR TITLE
feat: warn users when using new sbt-dependency-graph annotation

### PR DESCRIPTION
**Pull request by mika-bar**
- [X] Ready for review
- [X] Follows CONTRIBUTING rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?
Currently our scala SnykSbtPlugin script can't handle the new naming for sbt-dependency-graph plugin
which is used on sbt versions 1.4+
We should warn users when they use it in order for them to use the old annotation, so that the new plugin inspect will be used.

### CODE REVIEW

File: lib/index.ts [hunk 1]
```diff
@@ -276,3 +282,14 @@ export function buildArgs(
 
   return args;
 }
+
+function buildNewSbtDepGraphWarning() {
+  return (
+    '\n\n' +
+    'We currently do not support the new `addDependencyTreePlugin` annotation for the `sbt-dependency-graph` plugin\n' +
+    'Instead, please add the following line to the plugins file using this command: \n' +
```
Comment (gitphill) [inline lib/index.ts:290]: `Instead, please add the following line to the plugins file:

**Review by gitphill — DISMISSED** [review]

File: test/fixtures/simple-app-sbt-1.4.0/build.sbt [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ThisBuild / organization     := "com.example"
+ThisBuild / organizationName := "example"
+
+lazy val root = (project in file("."))
+  .settings(
+    name := "simple-app",
+    libraryDependencies += scalaTest % Test
+  )
+
+// Uncomment the following for publishing to Sonatype.
```
Comment (jk05) [inline test/fixtures/simple-app-sbt-1.4.0/build.sbt:14]: not familiar with this file or whats going on here

**Review by jk05 — APPROVED** [review]

File: test/fixtures/simple-app-sbt-1.4.0/build.sbt (hunk 2 above)
Comment (mika-bar) [reply test/fixtures/simple-app-sbt-1.4.0/build.sbt:14]: this is a sbt project fixture which manifest' is a `build.sbt` file, as the sbt framework demands. 
we test those test cases on our unit tests along the code base.

File: lib/index.ts [hunk 3]
```diff
@@ -276,3 +282,14 @@ export function buildArgs(
 
   return args;
 }
+
+function buildNewSbtDepGraphWarning() {
```
[the remark this answers is not in the document]
Comment (mika-bar) [reply lib/index.ts:286]: yes but very long and messy when introduced through the outer function. 
also, keeping it at a similar pattern to the other message function used on this file.

**Pull request merged**